store/postgres/src/primary.rs

```diff
@@ -626,6 +627,22 @@ mod queries {
             .transpose()
     }
 
+    pub(super) fn assignment_status(
+        conn: &PgConnection,
+        site: &Site,
+    ) -> Result<Option<(String, bool)>, StoreError> {
```

Comment:
It would be clearer if this returned a `Option<(NodeId, bool)>`; if the conversion from `String` to `NodeId` fails, you can just return a `constraint_violation!` since that shouldn't happen.